federator/src/lib/Federator.ts

```diff
@@ -1,52 +1,55 @@
-const web3 = require("web3");
-const fs = require("fs");
-const TransactionSender = require("./TransactionSender");
-const CustomError = require("./CustomError");
-const BridgeFactory = require("../contracts/BridgeFactory");
-const FederationFactory = require("../contracts/FederationFactory");
-const AllowTokensFactory = require("../contracts/AllowTokensFactory");
-const utils = require("./utils");
-const metrics = require("./MetricCollector");
-const typescriptUtils = require("./typescriptUtils");
-module.exports = class Federator {
-  constructor(config, logger, metricCollector, Web3 = web3) {
+import { Logger } from 'log4js';
+import { Config } from './config';
+import { MetricCollector } from './MetricCollector';
+
+import web3 from 'web3';
+import fs from 'fs';
+import TransactionSender from './TransactionSender';
+import CustomError from './CustomError';
+import { BridgeFactory } from '../contracts/BridgeFactory';
+import { FederationFactory } from '../contracts/FederationFactory';
+import { AllowTokensFactory } from '../contracts/AllowTokensFactory';
+import utils from './utils';
+import * as typescriptUtils from './typescriptUtils';
+
+export class Federator {
```

Comment:
export as default to avoid the Federator.Federator thingy

Reply:
oww, nice!